.travis.yml

```diff
@@ -32,8 +32,8 @@ matrix:
   include:
```

Comment:
I don't think the matrix section is needed. The TEST_SUITE var can simply be defined in the env section above. https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix